nextcord/gateway.py

```diff
@@ -782,24 +806,32 @@ class DiscordVoiceWebSocket:
         _max_heartbeat_timeout: float
         thread_id: int
 
-    def __init__(self, socket, loop, *, hook=None):
-        self.ws = socket
-        self.loop = loop
-        self._keep_alive: Any = None
-        self._close_code = None
-        self.secret_key = None
-        self._hook = hook or getattr(self, "_hook", None) or getattr(self, "_default_hook")
-
-    async def _default_hook(self, *args):
+    def __init__(
+        self,
+        socket: DiscordClientWebSocketResponse,
+        loop: asyncio.AbstractEventLoop,
+        *,
+        hook: Optional[Callable] = None,
```

Comment:
https://github.com/nextcord/nextcord/pull/830/files#diff-d090b4de22b790cbf918f41845d109812ebb342752113453a347203ada62a6f4R865
Could this `Callable` be specified fully

Reply:
I'm not sure what the callable would look like specified fully. Could I have some assistance?

Reply:
https://github.com/nextcord/nextcord/pull/830/files#diff-d090b4de22b790cbf918f41845d109812ebb342752113453a347203ada62a6f4R821